packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -55,33 +55,33 @@
   "dependencies": {
     "@itwin/imodel-browser-react": "^0.12.1",
     "@itwin/itwinui-icons-react": "^1.7.0",
-    "@itwin/itwinui-react": "^1.29.2",
+    "@itwin/itwinui-react": "^1.33.0",
     "classnames": "^2.3.1",
     "cross-fetch": "^3.1.4",
     "immer": "^9.0.7",
     "simple-react-validator": "^1.6.1"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^3.0.0",
-    "@itwin/appui-layout-react": "^3.0.0",
-    "@itwin/appui-react": "^3.0.0",
-    "@itwin/build-tools": "^3.0.0",
-    "@itwin/components-react": "^3.0.0",
-    "@itwin/core-bentley": "^3.0.0",
-    "@itwin/core-common": "^3.0.0",
-    "@itwin/core-frontend": "^3.0.0",
-    "@itwin/core-geometry": "^3.0.0",
-    "@itwin/core-i18n": "^3.0.0",
-    "@itwin/core-markup": "^3.0.0",
-    "@itwin/core-orbitgt": "^3.0.0",
-    "@itwin/core-quantity": "^3.0.0",
-    "@itwin/core-react": "^3.0.0",
-    "@itwin/eslint-plugin": "^3.0.0",
-    "@itwin/itwinui-css": "^0.44.1",
-    "@itwin/presentation-common": "^3.0.0",
-    "@itwin/presentation-components": "^3.0.0",
-    "@itwin/presentation-frontend": "^3.0.0",
-    "@itwin/webgl-compatibility": "^3.0.0",
+    "@itwin/appui-abstract": "^3.0.1",
```

Comment:
was there something in 3.0.1 you needed? or would it have worked fine with 3.0.0?

Reply:
Was in the middle of this actually. Bump unneeded. Thought I had a transitive issue.